clean_data.py

```diff
@@ -43,30 +43,40 @@ def replace_mrn(df, key: str = 'ReplaceMe') -> pd.DataFrame:
 
 def classify_hearing_loss(df: pd.DataFrame):
   # Assign result to a temporary dataframe, since classifier creates lots of 
-  # temporary column names.
+  # temporary column names.  
```

Comment:
Looks like some extra spaces at the end of the line cause the diff to fail.  Can you remove them or revert this line?